dataprep/eda/basic/compute.py

```diff
@@ -1032,6 +1046,121 @@ def calc_heatmap(
 
 
+def calc_stats_num(
+    srs: dd.Series,
+) -> Tuple[Dict[str, Any], Dict[str, Any], Dict[str, Any]]:
+    """
+    Calculate stats from a numerical column
+
+    Parameters
+    ----------
+    srs
+        a numerical column
+
+    Returns
+    -------
+    Tuple[Dict[str, Any], Dict[str, Any], Dict[str, Any]]
+        Dictionaries that contain Overview, Quantiles and Descriptives
+    """
+
+    max_value = srs.max().compute()
+    min_value = srs.min().compute()
+    mean_value = srs.mean().compute()
+    srs_size = len(srs)
+    overview_dict = {
+        "Distinct Count": srs.nunique().compute(),
+        "Unique (%)": srs.nunique().compute() / srs.count(),
+        "Missing": srs.isna().sum().compute(),
+        "Missing (%)": 1 - (srs.count().compute() / srs_size),
+        "Infinite": ((srs == np.inf) | (srs == -np.inf)).sum().compute(),
+        "Infinite (%)": ((srs == np.inf) | (srs == -np.inf)).sum().compute() / srs_size,
+        "Mean": mean_value,
+        "Minimum": min_value,
+        "Maximum": max_value,
+        "Zeros": srs_size - np.count_nonzero(srs),
+        "Zeros (%)": 1 - np.count_nonzero(srs) / srs_size,
+        "Memory Size": srs.memory_usage().compute() / 1024,
+    }
+    quantiles_dict = {
+        "Minimum": min_value,
+        "5-th Percentile": srs.quantile(0.05).compute(),
+        "Q1": srs.quantile(0.25).compute(),
+        "Median": srs.quantile(0.5).compute(),
+        "Q3": srs.quantile(0.75).compute(),
+        "95-th Percentile": srs.quantile(0.95).compute(),
+        "Maximum": max_value,
+        "Range": max_value - min_value,
+        "IQR": srs.quantile(0.75).compute() - srs.quantile(0.25).compute(),
+    }
+    descriptives_dict = {
+        "Standard Deviation": srs.std().compute(),
+        "Coefficient of Variation": srs.std().compute() / mean_value,
+        "Kurtosis": kurtosis(srs, nan_policy="omit"),
+        "Mean": mean_value,
+        "Median Absolute Deviation": median_absolute_deviation(srs, nan_policy="omit"),
+        "Skewness": skew(srs, nan_policy="omit"),
+        "Sum": srs.sum().compute(),
+        "Variance": srs.var().compute(),
+    }
+    return (
+        {k: _format_values(k, v) for k, v in overview_dict.items()},
+        {k: _format_values(k, v) for k, v in quantiles_dict.items()},
+        {k: _format_values(k, v) for k, v in descriptives_dict.items()},
+    )
+
+
+def calc_stats_cat(srs: dd.Series) -> Dict[str, str]:
+    """
+    Calculate stats from a categorical column
+
+    Parameters
+    ----------
+    srs
+        a categorical column
+
+    Returns
+    -------
+    Dict[str, str]
+        Dictionary that contains Overview
+    """
+    overview_dict = {
+        "Distinct Count": srs.nunique().compute(),
+        "Unique (%)": srs.nunique().compute() / srs.count(),
+        "Missing": srs.isna().sum().compute(),
+        "Missing (%)": 1 - (srs.count().compute() / len(srs)),
+        "Memory Size": srs.memory_usage().compute() / 1024,
```

Comment:
Since I use my own method processing every value calculated, I would prefer get this done in the function rather than passing the value that are already multiplied by 100 in `calc_bar_pie`